flag.go

```diff
@@ -321,6 +322,18 @@ func (f *FlagSet) MarkShorthandDeprecated(name string, usageMessage string) erro
 	return nil
 }
 
+// MarkPrivate indicated that a flag is private in your program. It will
+// continue to function but will not show up in help or usage messages. Using
+// this flag will also print the given usageMessage.
+func (f *FlagSet) MarkPrivate(name string, usageMessage string) error {
```

Comment:
@eparis; yes thank you, that was some c/p left-over from the MarkDeprecated method. I've removed that, and updated the comment as well.